kubernetes-kit-starter/src/main/java/com/vaadin/kubernetes/starter/sessiontracker/SessionSerializer.java

```diff
@@ -334,7 +336,7 @@ private void handleSessionSerialization(String sessionId,
             }
         } catch (PessimisticSerializationRequiredException e) {
             getLogger().warn(
-                    "Optimistic serialization of session {} with distributed key {} cannot be completed "
+                    "Optimistic serialization of session {} with distributed key {} cannot be completed"
                             + " because VaadinSession lock is required. Switching to pessimistic locking.",
                     sessionId, clusterKey, e);
         } catch (NotSerializableException e) {
```

Comment:
If the exception is thrown by `checkUnserializableWrappers` it is pointless to print the stack trace. It also floods the logs at every request.
I propose to detect the case using `PessimisticSerializationRequiredException` subclass or adding a field to the class (e.g. `boolean critical`, true by default, false for UnserializableWrapper check)

Reply:
I created a subclass to handle this and changed the log level to debug in case of a wrapper is found.